Exiled.API/Enums/EffectType.cs

```diff
@@ -199,5 +199,25 @@ public enum EffectType
         /// Teleports the player to the pocket dimension and drains health until the player escapes or is killed.
         /// </summary>
         PocketCorroding,
+
+        /// <summary>
+        /// Reduces walking sound by 10%.
+        /// </summary>
+        SilentWalk,
+
+        /// <summary>
+        /// Makes you a marshmallow guy.
+        /// </summary>
+        Marshmallow,
```

Comment:
this need to be puted as Obsolete[] (and make sure to is still use in EffectTypeExtension just supress the Error)

it's will actually not working in game